website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -145,7 +145,7 @@ Workspaces may be configured to share their state with other workspaces within t
 
 ### Use Separate Agent Pools for Sensitive Workspaces
 
-[Terraform Cloud Agents](/cloud-docs/agents) can be shared across all workspaces within an organization or [scoped to specific workspaces](/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). If agent pools are shared among multiple workspaces, a malicious actor in one of those workspaces could exfiltrate the agent’s API token, access private resources from the perspective of the agent, or modify the agent’s environment, potentially impacting other workspaces. For this reason, we creating separate agent pools for sensitive workspaces and using the agent scoping setting to restrict which workspaces can target each agent pool.
+[Terraform Cloud Agents](/cloud-docs/agents) can be shared across all workspaces within an organization or [scoped to specific workspaces](/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). If agent pools are shared among multiple workspaces, a malicious actor in one of those workspaces could exfiltrate the agent’s API token, access private resources from the perspective of the agent, or modify the agent’s environment, potentially impacting other workspaces. For this reason, we recommend creating separate agent pools for sensitive workspaces and using the agent scoping setting to restrict which workspaces can target each agent pool.
```

Comment:
```suggestion
You can share [Terraform Cloud Agents](/cloud-docs/agents) across all workspaces within an organization or [scope them to specific workspaces](/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). If multiple workspaces share agent pools, a malicious actor in one of those workspaces could exfiltrate the agent’s API token, access private resources from the perspective of the agent, or modify the agent’s environment, potentially impacting other workspaces. For this reason, we recommend creating separate agent pools for sensitive workspaces and using the agent scoping setting to restrict which workspaces can target each agent pool.
```

Small suggested word changes to avoid a passive voice style issue.

Reply:
LGTM!